src/avatar/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import cn from 'classnames';
+import img from './missing.png';
+
+import styles from './styles.css';
+
+export default ({ src = '', inline = '', defaultImage = img }) => {
+  const image = src || defaultImage;
```

Comment:
I agree, original had `defaultImage` variable - does it matter if we remove it when moving to evo?